static/app/views/organizationGroupDetails/groupEventDetails/index.tsx

```diff
@@ -31,53 +29,48 @@ type Props = RouteComponentProps<
   organization: Organization;
   project: Project;
   selection: PageFilters;
-};
+}
 
-type State = typeof OrganizationEnvironmentsStore['state'];
+export function GroupEventDetailsContainer(props: GroupEventDetailsProps) {
```

Comment:
Any chance you could pull out this FC refactor into it's own PR?

Reply:
To explain a bit why I suggested this --

It’s basically optimizing for writing code vs optimizing to reviewers of the changes. A nice thing about optimizing for review is that if you give people concise small things to look at (that perhaps can add up to a combination of bigger changes, but all separate PRs) you’ll tend to be able to get things merged in faster, vs if you have 2 semi related, but not obviously easy to review together changes, the reviewer has to do extra work to understand the two changes together.

So by pulling the FC conversion / test updates into a single PR we don't also need to think about the fix to the safe reflux store and can approve that quickly. Then after when looking at this safe reflux store fix we don't need to think about the FC/test changes.

Reply:
I dont mind that, as long as we enforce it across the rest of the org. The problem here was that fixing the reflux store breaks the tests, so had to have been updated too. I moved the fc to separate PR tho

Reply:
Separate PR for those looking for it https://github.com/getsentry/sentry/pull/33141

Reply:
@JonasBa it's not so much about enforcement, imo maybe this PR is on the borderline of it's small enough to not matter, but in general I also _try_ to adhere to the the rule of "_1 logical unit of work_" per PR / branch, since it does make reviewing way easier and faster. I know from a reviewing standpoint it makes me (at least) more inclined to review 😆

Reply:
I'm 200% on that train too :D I think we need to do this more and encourage folks to do it. I've personally avoided this process because of the state of our CI, merging tiny PR's was just a nightmare of just endlessly rerunning CI until it passed. But that has improved in this past week so I'll be sure we do that and encourage others to do the same :)